src/components/Disputes/Disputes.js

```diff
@@ -1,54 +1,115 @@
 import React, { useCallback, useMemo, useState } from 'react'
-import { Button, Header } from '@aragon/ui'
-
+import { Button, GU, Header, Tabs, Tag } from '@aragon/ui'
 import DisputeDetail from './DisputeDetail'
 import DisputeList from './DisputeList'
+import ANJIcon from '../../assets/anjButton.svg'
+import useDisputesSubscription from './hooks/useDisputesSubscription'
+import useJurorDraftQuery from './hooks/useJurorDraftQuery'
+
+const useSelectedDispute = disputes => {
+  const [selectedDisputeId, setSelectedDisputeId] = useState(-1)
+
+  const selectDispute = disputeId => setSelectedDisputeId(disputeId)
```

Comment:
I think it would be really nice to have the dispute ID in the URL, so that people can link to it, bookmark it etc. What do you think?

Reply:
Yeah agree!, is it ok for you if i do it in another PR? since the Dispute-detail PR changed some stuffs

Reply:
Yes, let’s do it after we get this PR merged!